### PR TITLE
Use github_changelog_generator for changelog

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,231 +1,334 @@
-## 0.10.0 / 2015-06-24
+# Change Log
 
-### Bug Fixes
+## [1.0.0.beta.2](https://github.com/test-kitchen/kitchen-ec2/tree/1.0.0.beta.2) (2016-02-26)
```
Comment (tyler-ball) [inline CHANGELOG.md]: Are you going to do the final `1.0.0` changelog after this is merged, or should that be in this PR?

**Comment by tyler-ball** [comment]
:+1:

**Comment by adamleff** [comment]
It seems as though this change is more than just a changelog / process change as there's some reserved-instance-related stuff in here?  I'm :+1: regardless, but perhaps we should update the title and/or add some text to the description of the PR for good record-keeping.

**Comment by jkeiser** [comment]
@adamleff it actually was just this, I had rebased from a non-master branch that I had made to fix rubocop so that change happened to get in here.

**Pull request merged**